clusters/build-clusters/01_cluster/openshift/ci-operator/registry-pull-credentials_secret_template.yaml

```diff
@@ -8,7 +8,11 @@ objects:
       openshift.io/description: |
         This secret contains the pull secrets for api.ci cluster and the local registry.
         It is used to pull images
-        * by the image clients in ci-operator pod: the local images synced by dptp-controller are eventually consistent only
+        * by image client: The DPTP-manager sets up lookupPolicy=true with some delay (in seconds). Before it is set,
```

Comment:
No, this has nothing to do with the dptp-controller-manager. The eventual consistency comes from the registry-replacer setting up replacements only hourly, not immediately when a dockerfile is changed. The replacement (sample: https://github.com/openshift/release/pull/12504/files#diff-69329224028311176ebfd8c08b29fb02R47) makes us use that image from the local cluster, without the replacement, it gets pulled from api.ci directly